### PR TITLE
(SIMP-4199) Update metadata.json for 6.1.0

### PR DESCRIPTION
**Pull request by trevor-vaughan**
This was missed in the final release

SIMP-4199 #close

### CODE REVIEW

**Comment by trevor-vaughan** [comment]
SIMP-4239 #close

File: metadata.json [hunk 1]
```diff
@@ -19,16 +19,324 @@
   ],
   "dependencies": [
     {
-      "name": "simp/compliance_markup",
+      "name": "camptocamp/kmod",
       "version_requirement": ">= 2.1.0 < 3.0.0"
     },
     {
-      "name": "simp/simp_options",
+      "name": "herculesteam/augeasproviders_apache",
```
Comment (jeefberkey) [inline metadata.json]: Is there code that you wrote to generate this list? Can you drop it in the Rakefile?

Comment (trevor-vaughan) [reply metadata.json]: @jeefberkey No, there is not

**Comment by lnemsick-simp** [comment]
* Values match the dependencies simp.spec for 6.1.0-0 tag.
* Would be advisable to remove mcollective, as it doesn't work

Because ranges are specified, this configuration has not been tested. I will manually verify a SIMP server can be bootstrapped with this set of module dependencies,  when the other environment modifications required are also made (e.g., proper setup of /etc/simp/environments....).

**Comment by lnemsick-simp** [comment]
Attached is a summary of the dependency analysis.  It lists the module dependencies from the following sources:
* The simp-simp_core metadata.json in the PR
* The 6.1.0-0 Puppetfile.tracking, which identifies the exact module versions used to build and test SIMP 6.1.0-0 ISOs
* The 6.1.0-0 simp.spec, which was used to build the simp RPM
* The results of a 'puppet module install' of the simp-simp_core meta module on 1/16/2018
[dependencies.xlsx](https://github.com/simp/simp-core/files/1639507/dependencies.xlsx)

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/simp-core/pipelines/16277694

**Comment by jeefberkey** [comment]
forge_install passed

**Pull request merged**